src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  margin: 20px 0;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: 300;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  outline: none;
+}
+
+.input-big {
+  height: 70px;
+  padding-left: 62px;
+  background-position: 26px 25px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input-small {
+  height: 42px;
+  padding-left: 33px;
+  background-size: 11px;
+  background-position: 13px 15px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input::placeholder {
+  font-weight: 400;
```

Comment:
I guess there should be 300 as well as in `.input` (as described in the task)
also, fix it in @font-face: set font-weight 300